Assets/SteamVR_Unity_Toolkit/Examples/Resources/Scripts/AutoRotation.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+using System.Collections;
+namespace VRTKX
```

Comment:
Before I decided to contribute to VRTK, I thought about extending VRTK (VRTKX), I will remove this namespace